academic_observatory_workflows/workflows/oa_web_workflow.py

```diff
@@ -1237,6 +1353,8 @@ def __init__(
         ror_dataset_id: str = "ror",
         settings_dataset_id: str = "settings",
         version: str = "v3",
+        conceptrecid: int = 6399462,
+        zenodo_host: str = "https://zenodo.org",
```

Comment:
Could you add descriptions for these params to the docstring just below? It is a little bit unclear what the 'conceptrecid' is

Reply:
Thanks Aniek, yeah sure.